workspaces/ui-v2/src/pages/diffs/ReviewDiffPages.tsx

```diff
@@ -32,7 +33,8 @@ export function DiffReviewPages(props: any) {
   //dependencies
   const diff = useDiffsForCapture(boundaryId, diffId);
   const allRequestsAndResponsesOfBaseSpec = useAllRequestsAndResponses();
-  const allEndpointsOfBaseSpec = useEndpoints();
+  useFetchEndpoints();
```

Comment:
This is used to initiate the fetch request. This request in particular could be globally triggered (i.e. anywhere since it's used in every page) but for now i think it's fine to leave it in the top level of each page's entry point.

Reply:
Also, I think it's a little weird since it could be easy to "forget" to initiate a request (but fairly obvious since the loading state will never complete) - but we _want_ this level of decoupling and being explicit here increases the control we have